lectures/inequality.md

```diff
@@ -616,51 +619,11 @@ We will use US data from the {ref}`Survey of Consumer Finances<data:survey-consu
 df_income_wealth.year.describe()
 ```
 
-This code can be used to compute this information over the full dataset.
+{download}`This notebook <_static/lecture_specific/inequality/data.ipynb>` can be used to compute this information over the full dataset.
```

Comment:
How will this look in the printed version?

Reply:
@jstac this used to render in the `pdf` as a link but it looks like that has changed in sphinx. I'll open a `meta` issue as we use these in a few cases. 

I have changed this to a link to github which you can download the notebook (or just view it).